Versionize.Tests/WorkingCopyTests.cs

```diff
@@ -93,6 +93,46 @@ public void ShouldExitIfWorkingCopyIsDirty()
         _testPlatformAbstractions.Messages[0].ShouldBe($"Repository {_testSetup.WorkingDirectory} is dirty. Please commit your changes:\nNewInIndex: hello.txt");
     }
 
+    [Fact]
+    public void ShouldExitIfChangelogPathsPointingToSameLocation()
+    {
+        var projects = new[]
+        {
+            new ProjectOptions
+            {
+                Name = "Project1",
+                Path = "project1",
+                Changelog = ChangelogOptions.Default with
+                {
+                    Header = "Project1 header",
+                    Path = "../docs"
+                }
+            },
+            new ProjectOptions
+            {
+                Name = "Project2",
+                Path = "project2",
+                Changelog = ChangelogOptions.Default with
+                {
+                    Header = "Project2 header",
+                    Path = "../docs"
+                }
+            }
+        };
+
+        var config = new FileConfig
+        {
+            SkipDirty = true,
+            Projects = projects
+        };
+        var json = Newtonsoft.Json.JsonConvert.SerializeObject(config);
+        File.WriteAllText(Path.Join(_testSetup.WorkingDirectory, ".versionize"), json);
+
+        Should.Throw<CommandLineExitException>(() =>
+            ConfigProvider.GetSelectedOptions(_testSetup.WorkingDirectory, CliConfig.Create(new McMaster.Extensions.CommandLineUtils.CommandLineApplication())));
```

Comment:
Because the line is quite long, will you please move the config creation to a separate variable?

```
var cliConfig = CliConfig.Create(new McMaster.Extensions.CommandLineUtils.CommandLineApplication());
```